packages/database/src/migrations/20210124014513-UpdateLaosSchoolsDevPartnersMapOverlays-modifies-data.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+import { arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+const MAP_OVERLAY_IDS = [
+  'Laos_Schools_Dev_Partner_SchDP_AEAL',
+  'Laos_Schools_Dev_Partner_SchDP_CRS',
+  'Laos_Schools_Dev_Partner_SchDP_HII',
+  'Laos_Schools_Dev_Partner_SchDP_Plan',
+  'Laos_Schools_Dev_Partner_SchDP_RtR',
+  'Laos_Schools_Dev_Partner_SchDP_UNIC',
+  'Laos_Schools_Dev_Partner_SchDP_WB',
+  'Laos_Schools_Dev_Partner_SchDP_WC',
+  'Laos_Schools_Dev_Partner_SchDP_WFP',
+  'Laos_Schools_Dev_Partner_SchDP_WR',
+  'Laos_Schools_Dev_Partner_SchDP_WV',
+];
```

Comment:
It's easier to update `Yes/No` to `1/0` for these map overlays separately because the presentationOptions references `Yes/No` multiple times in different ways